api/app/utils/errors.py

```diff
@@ -65,3 +65,5 @@
     {'message': 'Offer has not been accepted'}, 409)
 APPLICATIONS_CLOSED = (
     {'message': 'Applications are now closed'}, 403)
+NO_ACCOMMODATION_TRAVEL_AWARD = (
+    {'message': 'No template found because your offer has neither accommodation nor travel award allocation'}, 404)
```

Comment:
There is a template if you have neither.
You still get an invitation letter even if they're not sponsoring travel or accomodation :P